pages/article.js

```diff
@@ -39,6 +68,24 @@ class Articles extends Component {
         } else {
           return <Layout {...this.props}>
             <Article article={article} />
+            <Mutation mutation={this.newComment}
+              update={(cache, { data: { newComment } }) => {
```

Comment:
I would get the mutation working without this update function first. Can add it in after. The mutation should add the comment. But you'll need to refresh the page to see it at first